src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkUnix.cs

```diff
@@ -64,8 +68,7 @@ protected override string FindFramework (string runtimePath)
 		protected override string FindBin (string runtimePath)
 		{
 			string binPath = Path.GetFullPath (Path.Combine (runtimePath, "..", "..", "bin"));
-			Console.WriteLine (binPath);
-			if (File.Exists (Path.Combine (binPath, "generator")))
+			if (File.Exists (Path.Combine (binPath, GeneratorExe)))
```

Comment:
This change confuses me. We're trying to validate the `bin` directory, but there are no circumstances in which `$prefix/bin/generator.exe` will exist, so why check for `GeneratorExe`? Should this be `GeneratorScript`?